Maquetado/scss/grid.scss

```diff
@@ -63,6 +67,15 @@
   text-align: center;
   width: 220px;
 
+  @media screen and (min-width: 1024px) {
```

Comment:
this styles are defined previously in lines 63 and 64, just add the media queries for small resolutions